recipes/cifkit/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "cifkit" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: da6ee47508c4f03f1636f0e55e95bfb697e89886b8b4ff45a0011bfca4b4b6be
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools
+    - pip
+
+  run:
+    - python >=3.10
+    - gemmi
+    - matplotlib
```

Unless you need the qt backend explicitly, this is recommended for a lighter package.
```suggestion
    - matplotlib-base
```